spec/helpers/view_helper_spec.rb

```diff
@@ -20,4 +20,16 @@
       end
     end
   end
+
+  describe "#subject_is_selected?" do
```

Comment:
If we move the helpers these should move too.

Reply:
Done